src/diffusers/loaders/unet.py

```diff
@@ -816,7 +823,11 @@ def _convert_ip_adapter_attn_to_diffusers(self, state_dicts, low_cpu_mem_usage=F
 
         # set ip-adapter cross-attention processors & load state_dict
         attn_procs = {}
+        lora_dict = {}
```

Comment:
let's make a separate loop to create `lora_dict`
```suggestion
        lora_dicts = []
        for state_dict in state_dict:
            key_id = 0
            lora_dict = []
            for name in for name in self.attn_processors.keys():
                if f"{key_id}.to_k_lora.down.weight" in state_dict["ip_adapter"]:
                        lora_dict.update(...)
                        lora_dict.update(...)
                        ...
                        key_id += 1
            lora_dicts.append(lora_dict)
```

Reply:
ok